target_stitch/__init__.py

```diff
@@ -139,7 +139,11 @@ def send(self, data):
         '''Send the given data to Stitch, retrying on exceptions'''
         url = self.stitch_url
         headers = self.headers()
-        response = self.session.post(url, headers=headers, data=data)
+        ssl_verify = True
+        if os.environ.get("TARGET_STITCH_SSL_VERIFY") == 'false':
+            ssl_verify = False
+
+        response = self.session.post(url, headers=headers, data=data, verify=ssl_verify)
```

Comment:
This concerns me a little bit.  I don't think verify is a t/f argument.  I think its either false, or its the path to the certificate store.  And when verify isn't provided the default certificate store is used.

http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification

Reply:
Its a little odd, but the default is in fact `True` despite the documentation:

https://github.com/requests/requests/blob/master/requests/sessions.py#L372-L373

Looks like proper values would be `True`, `False`, or a string pointing to a certs path:

```
>>> import requests
>>> foo = requests.Session()
>>> foo.verify
True
```